zh_cn.lang

```diff
@@ -299,8 +284,10 @@ gui.xaero_sort_name=名称
 gui.xaero_sort_symbol=首字母
 gui.xaero_sort_distance=距离
 gui.xaero_box_cave_maps=一个可以显示地面之下的地图模式，主要用来显示地底的洞穴和建筑内部。\n 该参数指的是在你的头顶上的“正方形”范围内有固实方块遮挡时启用洞穴模式。
+gui.xaero_box_cave_maps2=一个可以显示地面之下的地图模式，主要用来显示地底的洞穴和建筑内部。\n 该参数指的是在你的头顶上的“正方形”范围内有固实方块遮挡时启用洞穴模式。固实方块不要求在同一个 Y 轴高度。
```

Comment:
Does not seem correct when I auto-translate it back, but it could be a mistake in the auto-translation.
"This parameter refers to enabling cave mode when there is a solid block in the "square" above your head."

Multiple solid blocks are supposed to form a 2D square, but this makes it seem like only a single one is needed.